ui/src/kapacitor/components/RuleMessage.tsx

```diff
@@ -1,22 +1,23 @@
-import React, {Component} from 'react'
-import PropTypes from 'prop-types'
+import React, {Component, ChangeEvent} from 'react'
 
 import RuleMessageText from 'src/kapacitor/components/RuleMessageText'
 import RuleMessageTemplates from 'src/kapacitor/components/RuleMessageTemplates'
 import {ErrorHandling} from 'src/shared/decorators/errors'
 
+import {AlertRule} from 'src/types'
+
+interface Props {
+  rule: AlertRule
+  ruleActions: any
```

Comment:
We might want a type here, or at least a not to go back once the kapacitor rule actions are typed.

Reply:
thanks! i missed that